modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/notification/DDMFormEmailNotificationSender.java

```diff
@@ -476,14 +476,14 @@ protected String getUserName(
 	}
 
 	protected String getViewFormEntriesURL(
-			DDMFormInstance ddmFormInstance, ThemeDisplay themeDisplay)
+			DDMFormInstance ddmFormInstance, ServiceContext serviceContext)
```

Comment:
@antonio91dev please order the parameters to be consistent with the other methods `getEmailBody`, `createTemplate`, `createMailMessage` and `populateParameters `.